stdlib/websub/src/main/ballerina/src/websub/subscriber_service_endpoint.bal

```diff
@@ -94,21 +94,30 @@ public type Listener object {
             boolean subscribeOnStartUp = internal:toBoolean(strSubscribeOnStartUp);
 
             if (subscribeOnStartUp) {
-                string resourceUrl = <string>subscriptionDetails["resourceUrl"];
-                string hub = <string>subscriptionDetails["hub"];
-                string topic = <string>subscriptionDetails["topic"];
-
-                var clientConfig = trap <http:ClientEndpointConfig>subscriptionDetails["subscriptionClientConfig"];
-                http:ClientEndpointConfig? subscriptionClientConfig =
-                                                    clientConfig is http:ClientEndpointConfig ? clientConfig : ();
-
-                if (hub == "" || topic == "") {
-                    if (resourceUrl == "") {
-                        log:printError(
-                            "Subscription Request not sent since hub and/or topic and resource URL are unavailable");
-                        return;
-                    }
-                    var discoveredDetails = retrieveHubAndTopicUrl(resourceUrl, subscriptionClientConfig);
+                string? resourceUrl = ();
+                string? hub = ();
+                string topic;
+
```

Comment:
Shall we avoid these newlines?

Reply:
Seemed  a bit cluttered without this new line. Removed the one after the if block.

Fixed the rest. Thanks!